.storybook/preview.js

```diff
@@ -0,0 +1,27 @@
+import { makeStyles } from "@material-ui/core";
+import { LitmusThemeProvider } from "../src/theme/ThemeProvider";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(2),
+    display: "flex",
+    position: "relative",
+  },
+}));
+
+export const decorators = [
+  (Story) => {
+    const classes = useStyles();
+    return (
+      <LitmusThemeProvider>
+        <div className={classes.root}>
```

Comment:
Looks like this styling will always align the storybook components in a horizontal flex-row fashion, it might be better to have an optional prop here if the stories need to be flexed or not, for example, if you have two or more tab components it needs to be listed in a vertical manner, same for progress bars (looks better when vertically aligned in columns). The `display: "flex"` property can be altered with an optional param which stays row aligned by default .

Reply:
The global decorators have `display: "flex"`. However, where the user requires to have `display: "inline-block"` then the user can have local decorator to override it. Don't know if decorators can have a single boolean prop.

Reply:
https://storybook.js.org/docs/react/writing-stories/decorators Reference